WalkingChallenge/ProfileViewController.swift

```diff
@@ -442,7 +442,7 @@ class ProfileViewController: UIViewController {
   }
 
   func configureApp() {
-    let configurationView = ConfigurationViewController()
+    let configurationView = SettingsViewController()//ConfigurationViewController()
```

Comment:
Since ConfigurationViewController needed a lot of changes, I just went ahead and made a new Class. Removed the dead code.